src/select/mini.rs

```diff
@@ -434,16 +448,17 @@ where
 
     fn view_direction_all(ctx: &Context<Self>) -> Html {
         let suffix = if ctx.props().active {
-            String::new()
-        } else if let Some(suffix) = ctx.props().deactive_class_suffix.as_ref() {
-            suffix.as_ref().into()
+            std::borrow::Cow::Borrowed("")
         } else {
-            String::new()
+            ctx.props().deactive_class_suffix.as_ref().map_or_else(
+                || std::borrow::Cow::Borrowed("-deactive"),
+                |suffix| std::borrow::Cow::Owned(suffix.to_string()),
+            )
         };
         let (class, class_text, class_icon) = (
-            format!("mini-select-top-direction{suffix}"),
-            format!("mini-select-top-direction-text{suffix}"),
-            format!("mini-select-top-direction-icon{suffix}"),
+            classes!(format!("mini-select-top-direction{suffix}")),
+            classes!(format!("mini-select-top-direction-text{suffix}")),
+            classes!(format!("mini-select-top-direction-icon{suffix}")),
```

Comment:
We don’t need classes! here since we’re passing a single, already-formatted string